tests/sqllogictests/suites/stage/formats/parquet/parquet_field_types.test

```diff
@@ -179,6 +179,22 @@ copy into t2 from @data/parquet/json_string.parquet
 ----
 parquet/json_string.parquet 4 0 NULL NULL
 
+query 
+select * from t2 order by a
+----
+1 "7B226B31223A227631222C226B32223A227632227D"
```

Comment:
The data in `json_string.parquet` is a JSON string, can we treat it as a string type and convert it to variant type by `json_parser` function?

Reply:
The query plan for copy into is built in the `bind_copy_into_table_from_location` function, we can add a check source type is binary to deal with this case

Reply:
I don't think it's a wise behavior. Because we are assuming all LargeBinary is valid uft8 and even valid json strings.

A better choice would be changing the parqute column type to LargeUtf8 and let user call parse_json when using copy into.

Reply:
How about implementing `parse_json(binary)` ?

Reply:
We don't need to support `parse_json(binary)`, and it is ok for users to use utf8 type as the source of variant type. The documentation needs to be updated to add a note to remind users to make sure the types are correct. 
For this test, it would be a good idea to modify the `json_string.parquet` file to change the type from binary to utf8 to make sure it's the right type, copy into a binary type to a variant type should be not using case. @andylokandy

Reply:
Parquet files could be updated in next pr.